ee/tabby-db/src/smtp_creds.rs

```diff
@@ -0,0 +1,53 @@
+use anyhow::Result;
+use rusqlite::OptionalExtension;
+
+use crate::DbConn;
+
+pub struct SMTPInfo {
```

Comment:
Please follow the convention that using a `DAO` suffix.
https://github.com/TabbyML/tabby/blob/356d1b075101c9cfda9a2cf73840b26df56e94c4/ee/tabby-db/src/job_runs.rs#L7

DAO is short for Data Access Object: https://en.wikipedia.org/wiki/Data_access_object